pkgs/by-name/xr/xrbinder/package.nix

```diff
@@ -0,0 +1,52 @@
+# SPDX-FileCopyrightText: 2025 Red_M <nixpkgs-xr@red-m.net>
+#
+# SPDX-License-Identifier: MIT
+
+{
+  stdenv,
+  lib,
+  fetchFromGitLab,
+  cmake,
+  makeWrapper,
+  pkg-config,
+  SDL2,
+  imgui,
+  xorg,
+
+  xrSources,
+}:
+
+stdenv.mkDerivation rec {
+  inherit (xrSources.xrbinder)
+    pname
+    version
+    src
+    date
+    ;
+
+  buildInputs = [
+    (SDL2.overrideAttrs (old: {
+      dontDisableStatic = true;
+    }))
+    imgui
+  ];
+  nativeBuildInputs = [
+    cmake
+    makeWrapper
+    pkg-config
+    xorg.libX11.dev
```

Comment:
This should be in buildInputs and not have `.dev` as a suffix.

Note that nativeBuildInputs should only be dependencies that need to be executed during build time. Any library that is merely linked against belongs to buildInputs.